apps/kvittering-backend/src/core/email_service.py

```diff
@@ -0,0 +1,109 @@
+import logging
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from email.mime.application import MIMEApplication
+from typing import Dict, Any, Union, List, Optional
+import boto3
+from core.data_types import FormData
+from core.utils import get_current_date_string, format_amount
+
+logger = logging.getLogger(__name__)
+
+
+class EmailService:
+    """Class for handling email sending operations."""
+    
+    def __init__(self, ses_client=None):
+        """Initialize the EmailSender with an optional SES client.
+        
+        Args:
+            ses_client: An AWS SES client. If None, a default client will be created.
+        """
+        self.ses_client = ses_client or self._create_default_ses_client()
+    
+    @staticmethod
+    def _create_default_ses_client():
+        """Create and return a default Amazon SES client."""
+        try:
+            return boto3.client('ses', region_name="eu-north-1")
+        except Exception as e:
+            logger.error(f"Failed to create SES client: {str(e)}")
+            raise Exception(f"Could not initialize email service: {str(e)}")
+    
+    @staticmethod
+    def get_formatted_text(form: FormData) -> str:
```

Comment:
Would be cool we if we could use our emailing template service here in the future

(That one doesn't have code for attachments at the moment so I don't think it's possible to use right now)

Reply:
Yeah I considered using brevduen but no attachments as you say, and it was quicker to just fix it myself here